doc/release-process.md

```diff
@@ -120,11 +129,11 @@ necessary, but do NOT push these commits to the `version-X.Y.Z` branch on the
 upstream repository; they will be included in the release branch that will be
 created in the next step.
 
-### Create the release candidate branch
+### Create the release branch
 
-Run the release script to create the first release candidate. This will create
-a branch based upon the specified commit ID, then commit standard automated
-changes to that branch locally:
+Run the release script to create the release commit. This will create a branch
+based upon the specified commit ID, then commit standard automated changes to
+that branch locally:
```

Comment:
This now reads confusingly to me, because my brain tries to interpret "release commit" and "specified commit ID" as being the same thing, which they are not. And in any case, the release script creates multiple release commits.

Given the heading of this section, I think this should be:
```suggestion
Run the release script to create the release branch. This will create a branch
based upon the specified commit ID, then commit standard automated changes to
that branch locally:
```